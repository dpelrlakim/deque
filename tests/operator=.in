newDefault dq1
= dq2 dq1
push_front dq1 0
= dq2 dq1
newInit dq1
push_back dq1 -10
push_back dq1 10
push_back dq1 372
= dq2 dq1
pop_front dq1
pop_front dq1
pop_back dq1
= dq2 dq1
newDefault dq3
= dq1 dq3
