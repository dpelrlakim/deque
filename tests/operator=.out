dq1:
dq1:
dq2:
dq1: 0
dq2:
dq1: 0
dq2: 0
dq1: 2 3 5 7 11 13 17 19
dq2: 0
dq1: 2 3 5 7 11 13 17 19 -10
dq2: 0
dq1: 2 3 5 7 11 13 17 19 -10 10
dq2: 0
dq1: 2 3 5 7 11 13 17 19 -10 10 372
dq2: 0
dq1: 2 3 5 7 11 13 17 19 -10 10 372
dq2: 2 3 5 7 11 13 17 19 -10 10 372
dq1: 3 5 7 11 13 17 19 -10 10 372
dq2: 2 3 5 7 11 13 17 19 -10 10 372
dq1: 5 7 11 13 17 19 -10 10 372
dq2: 2 3 5 7 11 13 17 19 -10 10 372
dq1: 5 7 11 13 17 19 -10 10
dq2: 2 3 5 7 11 13 17 19 -10 10 372
dq1: 5 7 11 13 17 19 -10 10
dq2: 5 7 11 13 17 19 -10 10
dq1: 5 7 11 13 17 19 -10 10
dq2: 5 7 11 13 17 19 -10 10
dq3:
dq1:
dq2: 5 7 11 13 17 19 -10 10
dq3:
